app/services/where_parents_query.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+# AR query, that returns childs, having all (not any) of concrete parents
```

Comment:
TODO: # Can be explaned as: `ChildsModel.where(id: parent1.child_ids & parent2.child_ids & parentN.child_ids)`